src/combobox/style.ts

```diff
@@ -53,7 +53,7 @@ export const COMBOBOX_CSS = `
     position: relative;
     display: flex;
   }
-  .kuc-combobox__group__toggle__input {
+  input[type=text].kuc-combobox__group__toggle__input {
```

Comment:
in my opinion, we can use input.className
ex: input.kuc-combobox__group__toggle__input
input.kuc-combobox__group__toggle__input:focus
ref:
https://github.com/kintone-labs/kintone-ui-component/blob/master/src/base/datetime/date/style.ts#L2
https://github.com/kintone-labs/kintone-ui-component/blob/master/src/date-picker/style.ts#L63